### PR TITLE
cleaned up the code a little

### PR DESCRIPTION
**Pull request by MohamedKaid**
touched up the code, switched it to a list View, and stopped it from taking in an empty input

### CODE REVIEW

File: app/src/main/java/com/example/techstacksetup/List.java [hunk 1]
```diff
@@ -71,10 +72,13 @@ public void onSuccess(QuerySnapshot queryDocumentSnapshots) {
                             String name = user.getName();
                             //Data is a large string with a new line after every entry
                             data += "Name: " + name + "\n";
-                            nameArray.add(data);
+                            nameArray.add(name);
```
Comment (alimo4) [inline app/src/main/java/com/example/techstacksetup/List.java:75]: String data is no longer in use and can be removed

File: app/src/main/java/com/example/techstacksetup/MainActivity.java [hunk 2]
```diff
@@ -34,31 +35,46 @@ protected void onCreate(Bundle savedInstanceState) {
...
         nameButton.setOnClickListener(
                 new View.OnClickListener()
                 {
-                    public void onClick(View view)
-                    {
-                        if (SystemClock.elapsedRealtime() - mLastClickTime[0] < 10000){
-                            helloName.setText("Button cool down. Please do not spam the save button.");
-                            return;
+                    public void onClick(View view) {
+                        if (!helloNameInput.getText().toString().isEmpty()) {
```
Comment (alimo4) [inline app/src/main/java/com/example/techstacksetup/MainActivity.java]: This if statement isn't necessary if the if statement on line 73 is placed before here. Just add a return to that if statement.

**Review by alimo4 — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/example/techstacksetup/MainActivity.java (hunk 2 above)
Comment (MohamedKaid) [reply app/src/main/java/com/example/techstacksetup/MainActivity.java]: this if statement is for spamming the save button, the if statement on line 73 is if you try to save a blank name it won't allow you to continue

File: app/src/main/java/com/example/techstacksetup/List.java (hunk 1 above)
Comment (MohamedKaid) [reply app/src/main/java/com/example/techstacksetup/List.java:75]: I left all that there just incase we need to reference back to it in cap project

**Review by alimo4 — APPROVED** [review]

**Pull request merged**